how-to-extend/adapter-starterkit/src/test/resources/application.conf

```diff
@@ -0,0 +1,20 @@
+# Event bus address of the Custom Datasource
```

Comment:
I'm wondering...
Maybe we should re-use configuration from `conf` here?
On the other hand that might be easier to understand the tests for readers...
What do you think?

Reply:
I had exactly the same thoughts - use only one configuration. But decided keep it for tests.

Reply:
This test validates only adapter. In `conf` we validate full scenario - from DataBridgeKnot through HttpDataSourceAdapter.